src/main/kotlin/org/rust/ide/intentions/ExtractInlineModuleIntention.kt

```diff
@@ -23,9 +23,6 @@ class ExtractInlineModuleIntention : RsElementBaseIntentionAction<RsModItem>() {
     override fun getFamilyName() = "Extract inline module structure"
     override fun getText() = "Extract inline module"
 
-    // No intention preview because it creates new file
-    override fun getFileModifierForPreview(target: PsiFile): FileModifier? = null
```

Comment:
I changed it because disabling preview via `getFileModifierForPreview` only works for `LocalQuickFix` and not for `IntentionAction`